qiskit_ibm_runtime/runtime_job.py

```diff
@@ -655,3 +656,19 @@ def tags(self) -> List:
             Tags assigned to the job that can be used for filtering.
         """
         return self._tags
+
+    @property
+    def usage_estimation(self) -> Dict[str, Any]:
+        """Return the usage estimation infromation for this job.
+
+        Returns:
+            ``quantum_seconds`` which is the estimated running
+            time of the job in seconds.
+        """
+        if not self._usage_estimation:
+            response = self._api_client.job_get(job_id=self.job_id())
+            self._usage_estimation = {
+                "quantum_seconds": response.pop("estimated_running_time_seconds", None),
+            }
```

Comment:
Is this data always available upon first query? If not, it'd be set to `"quantum_seconds": None` and never updated again (since `self._usage_estimation` won't be False).

Reply:
Yes, can confirm it's always available on first query